aiida_common_workflows/workflows/relax/quantum_espresso/generator.py

```diff
@@ -21,24 +16,25 @@
 class QuantumEspressoRelaxInputsGenerator(RelaxInputsGenerator):
     """Input generator for the `QuantumEspressoRelaxWorkChain`."""
 
-    _default_protocol = 'moderate'
     _calc_types = {'relax': {'code_plugin': 'quantumespresso.pw', 'description': 'The code to perform the relaxation.'}}
-    _relax_types = {
-        RelaxType.ATOMS: 'Relax only the atomic positions while keeping the cell fixed.',
-        RelaxType.ATOMS_CELL: 'Relax both atomic positions and the cell.'
-    }
-    _spin_types = {SpinType.NONE: '....', SpinType.COLLINEAR: '....'}
-    _electronic_types = {ElectronicType.METAL: '....', ElectronicType.INSULATOR: '....'}
+    _relax_types = {relax_type: '...' for relax_type in RelaxType}
+    _spin_types = {SpinType.NONE: '...', SpinType.COLLINEAR: '...'}
+    _electronic_types = {ElectronicType.METAL: '...', ElectronicType.INSULATOR: '...'}
```

Comment:
Don't you want to specify any description? I know that at the moment there is no method that returns the description, however there might be in the future and anyway it is useful for developers. Maybe "relax types" are self explanatory, but the "electronic_types" are no so intuitive, therefore a little sentence would help.

Reply:
Sure, I am just wondering what the benefit of this description is. Won't all plugins pretty much write something like "Run a spin-unpolarized calculation" for `SpinType.NONE`? Are we expecting that there needs to be plugin specific information here?

Reply:
For `RelaxType` and `SpinType` yes. But `ElectronicType` I saw at least three situations so far: 1) keyword ignored, 2) change of occupation  (smearing), 3) only increase the kpoints. Therefore something like: "run calculation with smearing" would be already useful for me to understand what to expect.

Reply:
I have added descriptions for `EletronicType`  and `SpinType`, but don't really feel to add a description for all entries in `RelaxType` since we support all. Maybe this is something to discuss if the description here should be code gnostic or agnostic. If its the former, then I would understand, because then we would explain how we accomplish the generic task with the specific plugin.

Reply:
Ok, I will add to the list for the discussion in next meeting